test/socket.ini

```diff
@@ -1,7 +1,7 @@
 [build]
 name = "socket-runtime-javascript-tests"
-input = src
-script = node build.js
+copy = src
+copy_map = src/mapping.ini
```

Comment:
what happens when both are presented?

Reply:
they both do what you'd expect

Reply:
are you copying `src/mapping.ini` to the output dir for testing purposes?